guides/plugins/plugins/administration/adding-snippets.md

```diff
@@ -69,6 +70,23 @@ The same `$tc` helper function can be used in the templates to access translatio
 
 {% endraw %}
 
+Another feature of `$tc` is pluralization. Using a `|` in snippets, you can provide translations dependending on a number. The first part would be shown as the singular expression, while the second takes care of your plural cases.
+Let's have a look at this example of `"examplePluralization": "One Product | {n} Products"` with the following implementation:
+
+{% raw %}
+
+```twig
+{% block my_custom_block %}
+    <p>
+       {{ $tc('swag-example.nested.examplePluralization', products.length) }}
+    </p>
+{% endblock %}
+```
+
+{% endraw %}
+
+If you provide `1` as a parameter to `$tc()`, the text `One Product` would be rendered. Otherwise, if you provide e.g. `4` as a parameter you'll get `4 Products`.
```

Comment:
```suggestion
If you provide `1` as the parameter to `$tc()`, the text `One Product` would be rendered. For any other value greater than 1, the number itself is shown — for example, `4 Products`.
```

Reply:
Thank you for your suggestion. Since there are more than one parameters I specified it.